packages/quantic/force-app/solutionExamples/main/lwc/exampleSearch/e2e/pageObject.ts

```diff
@@ -0,0 +1,141 @@
+import type {Locator, Page} from '@playwright/test';
+
+export class SearchPageObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get searchbox(): Locator {
+    return this.page.locator('c-quantic-search-box');
+  }
+
+  get searchboxInput(): Locator {
+    return this.searchbox.getByTestId('search-box-input');
+  }
+
+  get refineToggle(): Locator {
+    return this.page.locator('c-quantic-refine-toggle');
+  }
+
+  get tabBar(): Locator {
+    return this.page.locator('c-quantic-tab-bar');
+  }
+
+  get results(): Locator {
+    return this.page.locator('c-quantic-result');
+  }
+
+  get sort(): Locator {
+    return this.page.getByTestId('search-page-sort');
+  }
+
+  get sortDropdownButton(): Locator {
+    return this.sort.locator('button');
+  }
+
+  async sortOptionButton(buttonName: string): Promise<Locator> {
+    return this.sort.getByRole('option', {name: buttonName});
+  }
+
+  async allSortLabelOptions() {
+    const options = await this.page
+      .locator('div[part="dropdown overlay"]')
+      .allInnerTexts();
+    const arrSort = options[0].split('\n');
+    return arrSort;
+  }
+
+  async getSortLabelValue() {
+    const arrSort = await this.allSortLabelOptions();
+    const sortLabelwithValueList = await Promise.all(
+      arrSort.map(async (item) => ({
+        label: item,
+        value: await this.page
+          .getByRole('option', {name: item})
+          .getAttribute('data-value'),
+      }))
+    );
+    return sortLabelwithValueList;
+  }
+
+  sortButton(buttonName: string): Locator {
+    return this.page.getByRole('option', {name: buttonName});
+  }
+
+  async clickSortButton(buttonName: string): Promise<void> {
+    await this.sortButton(buttonName).click();
+  }
+
+  get summary(): Locator {
+    return this.page.locator('c-quantic-summary');
+  }
+
+  // First in the facetsManager targets the one on the search page instead of the ones in the refine toggle modal.
```

Comment:
I don't know if it's a good idea to rely on that, perhaps with the filters call there could be something else we could filter on?

Or from a more specific parent-child DOM element chain?

If nothing comes to mind it's not that bad, it's just that this in the future has the potential of selecting something other than what we'd expect if the order in the page changes for example.